languages/csharp/exercises/concept/switch-statements/.meta/Example.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+
+// **** please do not modify the Manager class ****
+public class Manager
+{
+    public string Name { get; }
+    public string Activity { get; }
+
+    public Manager(string name, string activity)
+    {
+        this.Name = name;
+        this.Activity = activity;
+    }
+}
+
+// **** please do not modify the Incident enum ****
+public enum Incident
+{
+    RedCard,
+    YellowCard,
+    Fowl,
+    Injury
+}
+
+public static class PlayAnalyzer
+{
+    public static string AnalyzeOnField(int shirtNum)
+    {
+        string playerDescription = string.Empty;
+        switch (shirtNum)
+        {
+            case 1:
+                playerDescription = "goalie";
+                break;
+            case 2:
+                playerDescription = "left back";
+                break;
+            case 5:
+                playerDescription = "right back";
+                break;
+            case 3:
+            case 4:
+                playerDescription = "center back";
+                break;
+            case 6:
+            case 7:
+            case 8:
+                playerDescription = "midfielder";
+                break;
+            case 9:
+                playerDescription = "left wing";
+                break;
+            case 11:
+                playerDescription = "right wing";
+                break;
+            case 10:
+                playerDescription = "striker";
+                break;
+            default:
+                throw new ArgumentException();
+        }
+
+        return playerDescription;
```

Comment:
This is a personal preference of mine, so do with it what you like :)

```suggestion
        switch (shirtNum)
        {
            case 1:
                return "goalie";
            case 2:
                return "left back";
            case 5:
                return "right back";
            case 3:
            case 4:
                return "center back";
            case 6:
            case 7:
            case 8:
                return "midfielder";
            case 9:
                return "left wing";
            case 11:
                return "right wing";
            case 10:
                return "striker";
            default:
                throw new ArgumentException();
        }
```

Reply:
I think in general, I would return immediately  as you have (well,  actually I'd be using an initialized read-only dictionary) but I did it, as it is, to show the importance of `break`.  I will leave it as is.